src/components/Calculator.js

```diff
@@ -1,26 +1,41 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 
 let count = 1;
 class Design extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      data: ['AC', '+/-', '%', '/', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='],
-      classname: 'a',
-    };
+      total: '',
+      operation: '',
+      next: '0'
+    }
   }
 
   render() {
-    const el = this.state;
+    const { data } = this.props;
+    // const { newState } = this.props;
+    // const { obj } = this.props;
```

Comment:
- [ ] OPTIONAL
There are commented code blocks found in your code. Kindly endeavor to remove **all** commented code blocks from your code before pushing it to the repository so as to keep your code well organized.